helpers/lucet-toolchain-tests/signature.sh

```diff
@@ -0,0 +1,91 @@
+#! /bin/sh
+
+LUCET_DIR="."
+LUCET_TARGET_DIR="${LUCET_DIR}/target/debug"
+TMPDIR="$(mktemp -d)"
+
+if [ -d "${LUCET_DIR}/target/release" ]; then
+    LUCET_TARGET_DIR="${LUCET_DIR}/target/release"
+fi
+
+if ! command -v rsign >/dev/null; then
+    cargo install rsign2
```

Comment:
can we install this in the devenv image by default?

Reply:
Done. But once the next stable Rust is out, we can run the webassembly version with Lucet instead.